### PR TITLE
#162600112 Set test coverage threshold

### PR DESCRIPTION
**Pull request by luqmanoop**
#### What does this PR do?
Makes sure commits are only applied when test coverage is above 90%.

#### Description of Task to be completed?
Have test coverage threshold be 90%

#### How should this be manually tested?
```
git clone git@github.com:andela/balder-ah-backend.git

cd balder-ah-backend

npm install

npm run check-coverage
```

#### Any background context you want to provide?
The npm script `check-coverage` runs before every commit is applied, making sure we only commit changes that complies with our coverage threshold.

#### What are the relevant pivotal tracker stories?
[#162600112](https://www.pivotaltracker.com/story/show/162600112)

#### Screenshots
screen shot 2018-12-12 at 8 40 50 am

#### Questions:
N/A

### CODE REVIEW

File: src/db/config/config.js [hunk 1]
```diff
@@ -3,29 +3,18 @@ const dotenv = require('dotenv');
...
-    port: DB_PORT,
-    dialect: DB_DIALECT
+    use_env_variable: 'DATABASE_URL',
+    dialect
   },
   test: {
-    use_env_variable: 'DATABASE_URL',
-    dialect: 'postgres',
-    logging: LOGGING === 'true' && true
+    use_env_variable: 'DATABASE_URL_TEST',
```
Comment (frostyblok) [inline src/db/config/config.js:15]: Hi Luqman, I like how clean this looks now, but I need to understand what is going on in here. Could you explain what you did here?

Comment (luqmanoop) [reply src/db/config/config.js:15]: Sure thing. I have refactored the file from using five (5) config properties to just one. It's a waste of space and more setup for someone new to the codebase.

Now, all we have to do in `.env` is add these lines
```
DATABASE_URL=postgresql://[host]/[databaseName]
DATABASE_URL_TEST=postgresql://[host]/[testDatabaseName]
```

Comment (frostyblok) [reply src/db/config/config.js:15]: OK, that makes sense! Good job.

**Review by frostyblok — APPROVED** [review]

**Comment by waleCloud** [comment]
I am closing because we've agreed the PR is not needed currently, as the problem it solves doesn't have an impact on the project as discussed during the TTL interaction.

**Pull request closed without merging**